### ISSUE TITLE
Outdated URL in README

### ISSUE THREAD

**Issue by fuwwy** [itsfolf/Discord-Screenshare-Linux#18]
Hi @itsfolf, i'm pretty sure you were the previous owner of the "fuwwy" handle -
i've noticed that the URL in the repo still points to the old username, so I tried to create that repo to see what would happen (I expected GitHub to block it for security reasons, which it didn't) and therefore broke the redirect (which means the install url in the README is now broken)

This also means that there's technically nothing that would stop me from injecting malicious code (which I won't do obviously, but better safe than sorry)

tldr; you should update the install command in your README

**Comment by itsfolf** [comment]
the good old repo jacking, thx for the catch
